include/swift/AST/DiagnosticsSema.def

```diff
@@ -2311,8 +2311,8 @@ NOTE(declared_protocol_conformance_here,none,
      (Type, unsigned, Identifier, Identifier))
 
 ERROR(witness_unavailable,none,
-        "unavailable %0 %1 was used to satisfy a requirement of protocol %2",
-        (DescriptiveDeclKind, DeclName, Identifier))
+        "unavailable %0 %1 was used to satisfy a requirement of protocol %2%select{|: %3}3",
```

Comment:
@RAJAGOPALAN-GANGADHARAN Hello, I am new to Diagnostics system, what's the meaning of `|:` in this case?

Reply:
Hey @wongzigii `%select{|: %3}3` : This statement will check if there is a message passed to available attribute if yes then it will print `error: message` or else simply `error` (note there is no `:` ). You can read more about this from docs/Diagnostics.md. Hope this helps :)